src/ordereddict.jl

```diff
@@ -8,7 +8,11 @@ import Base: haskey, get, get!, getkey, delete!, push!, pop!, empty!,
              hash, eltype, KeyIterator, ValueIterator, convert, copy,
              merge
 
+"""
+    OrderedDict
 
+'OrderedDict's are simply dictionaries whose entries have a particular order. The order refers to insertion order, which allows deterministic iteration over the dictionary or set.
```

Comment:
I think the intention here was to have ticks (\`) instead of apostrophes (') so that `OrderedDict` is rendered as code/monospace